### PR TITLE
Fix #36: Use recipe creation date for ordering

### LINKED ISSUE #36: Show experiments in order in the dropdown
For example, the experiment with ID 1 would be shown first, 2 second, and so on. It might be easiest to do this on the back-end and have the API guarantee this order.

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

**Comment by openjck** [comment]
Do I understand correctly that this changes the meaning of the `date` property returned from the `datasets` API? Does the *Through [somedate]* phrasing in the header still make sense or would something else be preferable?

File: notebooks/import.py [hunk 1]
```diff
@@ -82,12 +88,16 @@ def create_dataset(exp, process_date):
...
+            (name, slug, created_at, date, display, import_start)
+        VALUES (%s, %s, %s, %s, %s, %s)
+        RETURNING id
+    '''
     params = [
-        experiments.get(exp, {}).get('name', ''),
+        experiment_name,
         'TMP-%s' % exp,  # Store initially with a temporary prefix.
+        experiment_created_at,
         process_date_dt,
```
Comment (robhudson) [inline notebooks/import.py]: @openjck The "Through..." date is the same -- it's the `process_date_dt` variable here. This just adds a new date which tracks the experiment creation date from normandy and sorts the `/datasets/` API by this date.

Comment (openjck) [reply notebooks/import.py]: Ah, okay. Thanks for the clarification.

**Comment by openjck** [comment]
Looks good, tests pass. r+

**Pull request merged**